hera_pspec/tests/test_bootstrap.py

```diff
@@ -0,0 +1,102 @@
+import unittest
+import nose.tools as nt
+import numpy as np
+from hera_pspec import bootstrap as boot
+
+class Test_bootstrap(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def runTest(self):
+        pass
+    
+    def test_fold_kparallel(self):
+        #fold_kparallel(pspec_list)
+        pass
+    
+    def test_collapse_over_lsts(self):
+        #collapse_over_lsts(pspec_list)
+        pass
+    
+    def test_group_baselines(self):
+        """
+        Test baseline grouping behavior.
+        """
+        # Generate example lists of baselines
+        bls1 = [(0,i) for i in range(1)]
```

Comment:
The original pspecdata tutorial notebook has tuples with 3 elements. I think the code works fine with 3-element tuples (right?), but all of this should be clarified.

Reply:
Yes, I'll add a comment in the tests. This code just shuffles tuples or lists of tuples -- it doesn't care if they are valid 3-element baseline tuples or not. I think we don't need a validity check at this point either, as all of that will be done on passing the baselines into `pspec()`